### PR TITLE
Remove conflicting cta

### PR DESCRIPTION
**Pull request by dangjoey**

### CODE REVIEW

**Comment by trulyronak** [comment]
ty for the fast work! would it be possible to make the arrow going down to be clickable (or  to bring the learn more button back w/ a diff color and have that button scroll the user down)
Screen Shot 2022-01-27 at 9 23 58 PM

**Comment by dangjoey** [comment]
> ty for the fast work! would it be possible to make the arrow going down to be clickable (or to bring the learn more button back w/ a diff color and have that button scroll the user down) Screen Shot 2022-01-27 at 9 23 58 PM

Done!

File: src/sections/home/Home.Hero.tsx [hunk 1]
```diff
@@ -25,6 +25,10 @@ const HomeHero: React.FC = () => {
     return undefined
   }, [scrolled])
 
+  const scrollDown = () => {
+    window.scrollBy(0, window.innerHeight)
```
Comment (trulyronak) [inline src/sections/home/Home.Hero.tsx:29]: would it be possible to make this scroll smooth (like how when you click on 'contact' it smoothly scrolls down

if that's not possible dw about it, i'll just approve the pr anyways

approved, ty for making short work of this!!

**Review by trulyronak — APPROVED** [review]
would it be possible to make this scroll smooth (like how when you click on 'contact' it smoothly scrolls down

if that's not possible dw about it, i'll just approve the pr anyways

approved, ty for making short work of this!!

**Comment by steets250** [comment]
I believe this is what I added to ComputingPaths for smooth anchor scroll: https://github.com/iamdustan/smoothscroll, idk if it works with the current way we're scrolling

**Comment by dangjoey** [comment]
Fixed!

**Pull request merged**